study.md

```diff
@@ -112,13 +123,16 @@ From a collection of galaxies, update the `name` attribute of the record with
 the `designation` attribute of "NGC 224" to "Andromeda".
 
 ```ruby
-# your answer here
+Galaxy.where(designation:'NGC 224').update_all(name:"Andromeda")
```

Comment:
Be careful with this when you only want to update one record because what you've written will update all records matching that attribute and value.